class/defaults.yml

```
@@ -116,6 +118,7 @@ parameters:
         rules: ${keycloak:monitoring:rules}
       postgresql:
         enabled: ${keycloak:database:builtin}
+        existingSecret: ${keycloak:database:existingSecret}
```

I just noticed that it's easy to overwrite the password:
```suggestion
        postgresqlPassword: '?{vaultkv:${customer:name}/${cluster:name}/keycloak/db-password}'
```
It's possible to override this even via the hierarchy, but one has to know this...

---

ah now I understand...
the string `'?{vaultkv:${customer:name}/${cluster:name}/keycloak/db-password}'` gets base64 encoded and put it like this:
```yaml
data:
  postgresql-password: P3t2YXVsdGt2OnQtc2lsZW50LXRlc3QtMTIzNC9jLWdyZWVuLXRlc3QtMTIzNC9rZXljbG9hay9kYi1wYXNzd29yZH0=
```
which is `?{vaultkv:t-silent-test-1234/c-green-test-1234/keycloak/db-password}`. But ArgoCD won't be able to read that...

---

So we need to keep it as is, if you don't have a better solution/workaround.